### DISCUSSION TITLE
Better initial map zoom

### DISCUSSION THREAD

**Discussion by ajpanton** [Freika/dawarich#484] [Ideas]
When opening the map, or changing the date range for which it displays data, my request is to zoom and pan the map so that it covers the whole range of movements displayed.

I can see the possibility that you have zoomed to a location and then browse through dates to find a time when something happened in that particular location. In that case you don't want it to reset the zoom again for each date change. For that reason I suggest the following:

- A "default zoom" boolean that is set to True when you load the map.
- Whenever "default zoom" is True, the map should fit all movement that is shown.
- If at any time the user manually zooms or pans the map, "default zoom" is immediately set to False. Changing the date range should not change this value.
- Implement a new button next to the zoom buttons that resets the zoom. This will set "default zoom" to True.

**Comment by ezvr** [comment]
I would also like to see this. Has this ever been reviewed?
